src/index.scss

```diff
@@ -533,7 +534,74 @@ PACKAGES
 		height: 330px;
 	}
 }
+#not-found-page {
```

Comment:
This will only style the `<article className="container" id='not-found-page'>` element, not the entire background of the page. This is fine for the rest of the sub-elements, but it leaves the background color of the page w/e the default was and the 404 image/copy/etc... as a floating gray element.

Need to ensure the background-color is coloring the entire background